src/hooks/useInterval/useInterval.mdx

```diff
@@ -0,0 +1,64 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta title="hooks/useInterval" />
+
+# `useInterval`
+
+The `useInterval` hook is a custom React hook that allows you to repeatedly call a function with a
+specified delay. It is useful for creating animations, timers, and other time-based effects in your
+React component.
+
+> Please note that browser timers may not be completely accurate. The delay between intervals that
+> you specify is the minimum amount of time that will pass before the callback function is called
+> again, but it may not be exactly that amount of time.
+
+## Reference
+
+```tsx
+function useInterval(callback: () => void, ms?: number, enabled = true): void;
```

Comment:
Do we want to at least describe _something_ about the `enabled` attribute?
Like what it does, and how it does it (e.g. does it finish executing the current pending interval, or does it cancel it).